packages/api/portfolioDrafts/submit/subscribe.ts

```diff
@@ -0,0 +1,13 @@
+import { SQSEvent } from "aws-lambda";
+
+/**
+ * Recieves SQS Events and logs the MessageBody
+ *
+ * @param event - The SQS Event, triggered when a message is sent to the queue
+ */
+export async function handler(event: SQSEvent) {
+  event.Records.forEach((record: any) => {
+    const { body } = record;
+    console.log(body);
+  });
```

Comment:
```suggestion
  event.Records.map((record) => record.body).forEach(console.log);
```

You don't have to do this but I just wanted to point it out as an option.

Reply:
![weird](https://user-images.githubusercontent.com/84199040/135882643-8f3821fc-c02e-4788-bc5d-4503839f61d1.PNG)
This actually logs it twice?

Reply:
Okay, I implemented the fix